sql/table.go

```diff
@@ -87,13 +91,21 @@ func makeTableDesc(p *parser.CreateTable) (structured.TableDescriptor, error) {
 				ColumnNames: d.Columns,
 			}
 			if d.PrimaryKey {
-				index.Name = structured.PrimaryKeyIndexName
+				if index.Name == "" {
```

Comment:
See the comment in table_test.go about being able to see the name of primary index.